app/db/connection.py

```diff
@@ -0,0 +1,104 @@
+
+
+import psycopg2
+from psycopg2 import pool
+from psycopg2.extras import RealDictCursor
+from contextlib import contextmanager
+
+# Database configuration
+DB_CONFIG = {
```

Comment:
Why is this duplicated (also in `__init__.py`)? Also it should get it from a secret file.